cli.js

```diff
@@ -2,14 +2,17 @@
 'use strict';
 var path = require('path')
 
-var windows = process.platform === 'win32'
+    var windows = process.platform === 'win32'
 var pathVarName = (windows && !('PATH' in process.env)) ? 'Path' : 'PATH'
 
-process.env[pathVarName] += path.delimiter + path.join(__dirname, 'node_modules', '.bin')
+process.env[pathVarName] +=
+    path.delimiter +
+    path.join(__dirname, 'node_modules', '.bin')
```

Comment:
Could you please revert this formatting? It does not seem to add anything in this case